launch_testing/launch_testing_examples/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>launch_testing_examples</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="aditya050995@gmail.com">aditya</maintainer>
```

Comment:
For maintenance reasons, I would assign the same maintainers as the other packages in this repo:

```
  <maintainer email="mabel@openrobotics.org">Mabel Zhang</maintainer>
  <maintainer email="sloretz@openrobotics.org">Shane Loretz</maintainer>
```

You can add an `<author>` tag for yourself!